content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst

```diff
@@ -0,0 +1,289 @@
+
+On the new |BOM|, switch to the :guilabel:`Components` tab. Modify the components list by changing
+the :guilabel:`Quantity` of existing components, add new components using the :guilabel:`Add a line`
+button, and remove components with the :guilabel:`🗑️ (trash)` icon.
+
+.. _plm/eco/example-keyboard:
+
+.. example::
+   In version two of the |BOM| for a keyboard, the component quantities are reduced, and an
+   additional component, `Stabilizers`, is added.
+
+   .. image:: engineering_change_orders/version-2-BOM.png
+      :align: center
+      :alt: Make changes to components by going to the new BoM with the *Revision* smart button.
+
+Once the changes are complete, navigate back to the |ECO| itself, by clicking `ECO00X` in the
+breadcrumbs in the top-left corner. On the |ECO| form, a new :guilabel:`BoM Changes` tab displays
+the differences between the current |BOM| and the new version.
+
+This ensures changes and tests are encapsulated in the revised |BOM| and do not affect the |BOM|
+currently used in production. That is, until the :ref:`changes are applied <plm/eco/apply-changes>`.
+
+.. example::
+   View the summary of the differences between the current and revised keyboard |BOMs| in the
+   :guilabel:`BoM Changes` tab of the |ECO|.
+
+   .. image:: engineering_change_orders/BoM-changes.png
+      :align: center
+      :alt: View summary of component changes in the *BoM Changes* tab.
+
+Change operations
+=================
+
+To modify the operations in a |BOM|, click the :guilabel:`Revision` smart button on an active |ECO|
+to access the archived, new version of the |BOM|.
+
+In the new |BOM| version, switch to the :guilabel:`Operation Changes` tab, and make changes to the
+operations, by selecting each operation line item, and opening the corresponding :guilabel:`Open:
+Operations` pop-up window.
+
+Make changes to any of the fields in the pop-up window, and click :guilabel:`Save` once completed.
+
+Create new operations by clicking the :guilabel:`Add a line` button, and remove new operations by
+clicking the :guilabel:`Archive` button.
+
+.. note::
+   The :guilabel:`Operations` tab is *not* available by default. To enable it, navigate to
+   :menuselection:`Manufacturing app --> Configuration --> Settings`, and check the :guilabel:`Work
+   Orders` box.
+
+Once the changes are complete, navigate back to the |ECO| itself, by clicking `ECO00X` in the
+breadcrumbs in the top-left corner.
+
+On the |ECO| form, a new :guilabel:`Operation Changes` tab displays the differences between the
+current production |BOM| and the new version. Modifications to the |BOM| in an |ECO| will *not*
+affect the |BOM| used in production. That is, until the :ref:`changes are applied
+<plm/eco/apply-changes>`.
+
+.. _plm/eco/apply-changes:
+
+Apply changes
+=============
+
+After verifying the changes, move the |ECO| to a :ref:`verification stage <plm/eco/stage-config>`.
```

Comment:
Do you think this rewrite is sufficient? Otherwise, I'll think again about how to define this better.

> After verifying the changes, move the |ECO| to a :ref:`verification stage <plm/eco/stage-config>`,
which are stages that require approval before the revised changes can be applied to production
|BOM|.